src/backend/distributed/executor/multi_task_tracker_executor.c

```diff
@@ -879,13 +881,14 @@ TrackerConnectPoll(TaskTracker *taskTracker)
 			if (pollStatus == CLIENT_CONNECTION_BUSY_READ ||
 				pollStatus == CLIENT_CONNECTION_BUSY_WRITE)
 			{
-				uint32 maxCount = REMOTE_NODE_CONNECT_TIMEOUT / RemoteTaskCheckInterval;
```

Comment:
Wouldn't a Max(1, NodeConnectionTimeout / RemoteTaskCheckInterval) be better? Because this still can be 0?